x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -175,7 +199,48 @@ export const AlertsList: React.FunctionComponent = () => {
     }
   }
 
+  async function loadAlertsTotalStatuses() {
+    let alertsStatuses = {};
+    try {
+      AlertExecutionStatusValues.forEach(async (status: string) => {
+        const alertsTotalResponse = await loadAlerts({
+          http,
+          page: { index: 0, size: 0 },
+          searchText,
+          typesFilter,
+          actionTypesFilter,
+          alertStatusesFilter: [status],
+        });
+        setAlertsStatusesTotal({ ...alertsStatuses, [status]: alertsTotalResponse.total });
+        alertsStatuses = { ...alertsStatuses, [status]: alertsTotalResponse.total };
+      });
+    } catch (e) {
+      toastNotifications.addDanger({
+        title: i18n.translate(
+          'xpack.triggersActionsUI.sections.alertsList.unableToLoadAlertsStatusesInfoMessage',
+          {
+            defaultMessage: 'Unable to load alert statuses info',
+          }
+        ),
+      });
+    }
+  }
+
   const alertsTableColumns = [
+    {
+      field: 'executionStatus',
+      name: i18n.translate(
+        'xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.statusTitle',
+        { defaultMessage: 'Status' }
+      ),
+      sortable: false,
+      truncateText: false,
+      'data-test-subj': 'alertsTableCell-status',
+      render: (executionStatus: AlertExecutionStatus) => {
+        const healthColor = getHealthColor(executionStatus.status);
+        return <EuiHealth color={healthColor}>{executionStatus.status}</EuiHealth>;
+      },
+    },
```

Comment:
We have gone back and forth a bit on this.  I like it on the left since I think users will want to see this information first.  Perhaps we will find out more after it's used more, but this feels right to me.